e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -75,6 +77,127 @@ describe('Integrations', () => {
         });
     });
 
+    it('MM-T678 /code', () => {
+        loginAndVisitChannel(user1, testChannelUrl);
+
+        // # Use "/code"
+        cy.postMessage('/code 1. Not a list item, **not bolded**, http://notalink.com, ~off-topic is not a link to the channel.');
+
+        // * Verify that that markdown isn't rendered
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#post_${postId}`).find('.user-popover').should('have.text', user1.username);
+            cy.get(`#postMessageText_${postId}`).get('code').should('have.text', '1. Not a list item, **not bolded**, http://notalink.com, ~off-topic is not a link to the channel.');
+        });
+    });
+
+    it('MM-T679 /echo', () => {
+        loginAndVisitChannel(user1, testChannelUrl);
+
+        // # Type "/echo test 3"
+        cy.postMessage('/echo test 3');
+
+        // * Verify that that post is not shown after 1 second
+        cy.wait(TIMEOUTS.ONE_SEC);
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#postMessageText_${postId}`).should('have.not.text', 'test');
+        });
+
+        // * Verify that that test 3 is posted after 3 seconds
+        cy.wait(TIMEOUTS.TWO_SEC);
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#post_${postId}`).find('.user-popover').should('have.text', user1.username);
+            cy.get(`#postMessageText_${postId}`).should('have.text', 'test');
+        });
+    });
+
+    it('MM-T680 /help', () => {
+        loginAndVisitChannel(user1, testChannelUrl, {
+            onBeforeLoad: (win) => {
+                cy.stub(win, 'open');
+            },
+        });
+
+        // # Type "/help"
+        cy.postMessage('/help');
+
+        // * Verify that a new tag opens
+        cy.window().its('open').should('be.called');
```

Comment:
I'd suggest to expand verification here like
```suggestion
        cy.window().its('open').should('have.been.calledWithMatch', 'https://about.mattermost.com/default-help/');
```

Reply:
Good suggestion :+1: